### ISSUE TITLE
Allow post processing to work without being on a camera

### ISSUE THREAD

**Issue by ScoutWozniak** [Facepunch/sbox-issues#6531]
### For?

S&Box

### What can't you do?

Currently while I am testing out creating a new addon map, I'm experiencing the limitation of having control over the post processing effects.  While being able to edit the ones already built into the game is another discussion, at the very least I would appreciate having the ability to create new pp effect components in the scene.  This would allow for a better way for individual maps to control their look.

### How would you like it to work?

Being able to place a component in the scene and have it automatically find and connect to the main camera would be enough.

### What have you tried?

I've personally tried making my own post processing editor that mirrors the prior source 2 functionality but ultimately finishing it was beyond my current ability.

**Comment by DevulTj** [comment]
I agree with this. When we were working on HC1, it's something we were missing. We ended up having a camera prefab that a map can override instead - which is a bit shitty.

It'd be worth exposing post processing volumes, and have a "Is Global" bool for global overrides. What should we do when there's multiple of these, combine, override?

**Comment by ScoutWozniak** [comment]
It'd be really nice to have post processing volumes in general so I'll rename this issue to that as they seem to be the best solution for now.

I think a combination of combining them and overriding any shared effects between them would be a good idea.  Some games might require a specific post process effect (e.g. Opium's PSX style pp) so having that combine with a separate volume with tone mapping and color correction settings that exist per scene would be quite nice.

**Comment by Jammie3d** [comment]
In Unreal you can set the priority on a PPV, so if there's any overlap the highest priority number wins.

**Comment by Jammie3d** [comment]
The bloom amount on the Deathmatch camera was turned way up recently which made my blockout maps look like this:

![Image](https://github.com/user-attachments/assets/e5430dac-025e-4afc-8150-f4c179915f12)

Since it's an addon project I dont think there's much I can do to override any post processing variables.
I had to use duller dev textures and change the envmap probe texture to work around it, which isnt ideal.
